vnext/Shared/OInstance.cpp

```diff
@@ -432,7 +433,8 @@ void InstanceImpl::loadBundleSync(std::string &&jsBundleRelativePath) {
 bool isHBCBundle(const std::string &bundle) {
   static uint32_t constexpr HBCBundleMagicNumber = 0xffe7c3c3;
 
-  // Note:: Directly access the pointer to avoid copy/length-check. It matters as this string contains the bundle which can be potentially huge.
+  // Note:: Directly access the pointer to avoid copy/length-check. It matters as this string contains the bundle which
+  // can be potentially huge.
   // https://herbsutter.com/2008/04/07/cringe-not-vectors-are-guaranteed-to-be-contiguous/#comment-483
   BundleHeader *header = reinterpret_cast<BundleHeader *>(const_cast<char *>(&bundle[0]));
```

Comment:
I think you can get rid of the const_cast, if you reinterpret_cast to `const BundleHeader*`